code/aepcomposeui/src/main/java/com/adobe/marketing/mobile/aepcomposeui/aepui/AEPUI.kt

```diff
@@ -0,0 +1,22 @@
+package com.adobe.marketing.mobile.aepcomposeui.aepui
+
+import androidx.compose.runtime.mutableStateOf
+import com.adobe.marketing.mobile.aepuitemplates.AEPUITemplate
+import com.adobe.marketing.mobile.aepuitemplates.SmallImageTemplate
+import com.adobe.marketing.mobile.aepcomposeui.aepui.state.AEPUIState
+import com.adobe.marketing.mobile.aepcomposeui.aepui.state.SmallImageUIState
+
+
+/**
+ * Represents a UI component that can be rendered in the AepUI Engine. Binds a template data with a tracking observer.
```

Comment:
There is no AEPUIEngine anymore. Neither is there a tracking observer. Please update the description here